tests/vmi_multus_test.go

```diff
@@ -876,6 +877,47 @@ var _ = Describe("SRIOV", func() {
 		It("[test_id:3957]should connect to another machine with sriov interface over IPv6", func() {
 			pingThroughSriov("fc00::1/64", "fc00::2/64")
 		})
+
+		Context("VLANed SR-IOV network", func() {
+			var vlanedVMI1 *v1.VirtualMachineInstance
+			cidrVlaned1 := "192.168.0.1/24"
+			cidrVlaned2 := "192.168.0.2/24"
+			cidrNonVlaned := "192.168.0.3/24"
+
+			BeforeEach(func() {
+				createNetwork("sriov-vlan", tests.NamespaceTestDefault, sriovConfVlanCRD)
+				createNetwork("sriov-no-vlan", tests.NamespaceTestDefault, sriovConfNoVlanCRD)
+				vlanedVMI1 = createSriovVMI([]string{"sriov-vlan"})
+				// manually configure IP/link on sriov interfaces because there is
+				// no DHCP server to serve the address to the guest
+				configInterface(vlanedVMI1, "eth1", cidrVlaned1, "#")
```

Comment:
Using the console to configure the networking is pretty bad and full of flaky behavior.
I think you should work on creating VMI/s with cloud-init network configuration.

BTW, the prompt is wrong here, it should be "\\# " (although I think the `\\` is not needed).

Reply:
> Using the console to configure the networking is pretty bad and full of flaky behavior.
> I think you should work on creating VMI/s with cloud-init network configuration.

I agree. Though I think for the scope of this PR it is fine this way, and the work on
creating the VMIs (in this entire test suit) using cloud init configuration will be in a different PR.
is this acceptable?
> 
> BTW, the prompt is wrong here, it should be "\# " (although I think the `\\` is not needed).

Changed. but I think the `\\` is needed.